mne/report.py

```diff
@@ -553,7 +553,6 @@ def _build_html_slider(slices_range, slides_klass, slider_id):
     {{endif}}
     {{if comments is not None}}
         <br><br>
-        <div class="comments">
```

Comment:
something looks suspicious about this. Why don't your remove the closing `</div>` ?